appsdk/sdk.go

```diff
@@ -393,9 +391,24 @@ func (sdk *AppFunctionsSDK) Initialize() error {
 	sdk.webserver = webserver.NewWebServer(sdk.config, sdk.secretProvider, sdk.LoggingClient, mux.NewRouter())
 	sdk.webserver.ConfigureStandardRoutes()
 
+	sdk.LogStartUpMessage(dic, startupTimer)
+
 	return nil
 }
 
+// LogStartUpMessage logs the service startup message
+func (sdk *AppFunctionsSDK) LogStartUpMessage(dic *di.Container, startupTimer startup.Timer) {
+	sdk.LoggingClient.Info("Service dependencies resolved...")
+	sdk.LoggingClient.Info(fmt.Sprintf("Starting %s %s ", sdk.ServiceKey, internal.ApplicationVersion))
+
+	bootstrapConfig := container.ConfigurationFrom(dic.Get).GetBootstrap()
+	if len(bootstrapConfig.Service.StartupMsg) > 0 {
+		sdk.LoggingClient.Info(bootstrapConfig.Service.StartupMsg)
+	}
```

Comment:
Don't need to get from `dic`, use `sdk.config.Service.StartupMsg`

Reply:
Also this is already logged after the call to Initialize() returns. Need to remove that log message.